File: forge/src/main/scala/com/kotori316/fluidtank/forge/tank/ItemBlockVoidTankForge.scala
package com.kotori316.fluidtank.forge.tank

import java.util.function.Consumer

import com.kotori316.fluidtank.forge.render.RenderItemTank
import com.kotori316.fluidtank.tank.ItemBlockVoidTank
import net.minecraft.client.renderer.BlockEntityWithoutLevelRenderer
import net.minecraftforge.client.extensions.common.IClientItemExtensions

class ItemBlockVoidTankForge(b: BlockVoidTankForge) extends ItemBlockVoidTank(b) {

  override def initializeClient(consumer: Consumer[IClientItemExtensions]): Unit = {
    consumer.accept(new IClientItemExtensions {
      override def getCustomRenderer: BlockEntityWithoutLevelRenderer = RenderItemTank.INSTANCE
    })
  }

}

File: forge/src/main/scala/com/kotori316/fluidtank/forge/tank/ItemBlockVoidTankForge.java
package com.kotori316.fluidtank.forge.tank;

import com.kotori316.fluidtank.forge.render.RenderItemTank;
import com.kotori316.fluidtank.tank.BlockTank;
import com.kotori316.fluidtank.tank.ItemBlockVoidTank;
import net.minecraft.client.renderer.BlockEntityWithoutLevelRenderer;
import net.minecraftforge.client.extensions.common.IClientItemExtensions;

import java.util.function.Consumer;

public final class ItemBlockVoidTankForge extends ItemBlockVoidTank {
    public ItemBlockVoidTankForge(BlockTank b) {
        super(b);
    }

    @Override
    public void initializeClient(Consumer<IClientItemExtensions> consumer) {
        super.initializeClient(consumer);
        consumer.accept(new IClientItemExtensions() {
            @Override
            public BlockEntityWithoutLevelRenderer getCustomRenderer() {
                return RenderItemTank.INSTANCE();
            }
        });
    }
}
